admin/worker_v2.m.template

```diff
@@ -20,6 +20,10 @@ err_mess = [];
 exit_at_the_end = true;
 if isempty(which('herbert_init.m'))
     herbert_on();
+    try
+        horace_on();
```

Comment:
same comments as for `horace_on` in worker_for_tests above. We cannot assume Horace is installed or available running Herbert code.

Reply:
this is why its in try-catch block
if it is not installed, failure will be ignored. 
what other solution you can find?